sksurv/tree/tree.py

```diff
@@ -369,8 +369,7 @@ def predict(self, X, check_input=True):
         risk_scores : ndarray, shape = (n_samples,)
             Predicted risk scores.
         """
-        chf = self.predict_cumulative_hazard_function(X, check_input, return_array=True)
-        return chf.sum(1)
+        return self.predict_cumulative_hazard_function(X, check_input, return_array="sum")
```

Comment:
This changes all existing behaviour; an alternative would be to have an opt-in constructor flag so that callers need to opt-in to get the new behaviour.